scipy/sparse/linalg/matfuncs.py

```diff
@@ -841,7 +840,7 @@ def _ell(A, m):
     # The c_i are explained in (2.2) and (2.6) of the 2005 expm paper.
     # They are coefficients of terms of a generating function series expansion.
     choose_2m_m = scipy.special.comb(2*m, m, exact=True)
-    abs_c_recip = float(choose_2m_m * math.factorial(2*m + 1))
+    abs_c_recip = float(choose_2m_m) * float_factorial(2*m + 1)
```

Comment:
consider (but don't feel obliged to change)

technically this overflows at m=75 ((302 choose 151) * 151!)
Also choose_2m_m doesn't need exact=True (unless that's done for speed)

Reply:
>technically this overflows at m=75 ((302 choose 151) * 151!)

Well so long as `choose_2m_m` fits in a `float` (up to around `m=500`) , it will still give an `inf` instead of raising. Though now I look into this, `m` is never more than 13 in `_expm` so this is sort of irrelevant.

>Also choose_2m_m doesn't need exact=True (unless that's done for speed)

I tested and `exact=True` is around 20x faster(!) for small `m`.